eras/babbage/test-suite/cddl-files/babbage.cddl

```diff
@@ -0,0 +1,403 @@
+                   , ipv4 / null
+                   , ipv6 / null
+                   )
+single_host_name = ( 1
+                   , port / null
+                   , dns_name ; An A or AAAA DNS record
+                   )
+multi_host_name = ( 2
+                   , dns_name ; A SRV DNS record
+                   )
+relay =
+  [  single_host_addr
+  // single_host_name
+  // multi_host_name
+  ]
+
+pool_metadata = [url, pool_metadata_hash]
+url = tstr .size (0..64)
+
+withdrawals = { * reward_account => coin }
+
+update = [ proposed_protocol_parameter_updates
+         , epoch
+         ]
+
+proposed_protocol_parameter_updates =
+  { * genesishash => protocol_param_update }
+
+protocol_param_update =
+  { ? 0:  uint               ; minfee A
+  , ? 1:  uint               ; minfee B
+  , ? 2:  uint               ; max block body size
+  , ? 3:  uint               ; max transaction size
+  , ? 4:  uint               ; max block header size
+  , ? 5:  coin               ; key deposit
+  , ? 6:  coin               ; pool deposit
+  , ? 7: epoch               ; maximum epoch
+  , ? 8: uint                ; n_opt: desired number of stake pools
+  , ? 9: rational            ; pool pledge influence
+  , ? 10: unit_interval      ; expansion rate
+  , ? 11: unit_interval      ; treasury growth rate
+  , ? 14: [protocol_version] ; protocol version
+  , ? 16: coin               ; min pool cost
+  , ? 17: coin               ; ada per utxo byte
+  , ? 18: costmdls           ; cost models for script languages
+  , ? 19: ex_unit_prices     ; execution costs
+  , ? 20: ex_units           ; max tx ex units
+  , ? 21: ex_units           ; max block ex units
+  , ? 22: uint               ; max value size
+  , ? 23: uint               ; collateral percentage
+  , ? 24: uint               ; max collateral inputs
+  }
+
+transaction_witness_set =
+  { ? 0: [* vkeywitness ]
+  , ? 1: [* native_script ]
+  , ? 2: [* bootstrap_witness ]
+  , ? 3: [* plutus_script ]
+  , ? 4: [* plutus_data ]
+  , ? 5: [* redeemer ]
+  }
+
+plutus_script = bytes
+
+plutus_data =
+    constr<plutus_data>
+  / { * plutus_data => plutus_data }
+  / [ * plutus_data ]
+  / big_int
+  / bounded_bytes
+
+big_int = int / big_uint / big_nint
+big_uint = #6.2(bounded_bytes)
+big_nint = #6.3(bounded_bytes)
+
+constr<a> =
+    #6.121([* a])
+  / #6.122([* a])
+  / #6.123([* a])
+  / #6.124([* a])
+  / #6.125([* a])
+  / #6.126([* a])
+  / #6.127([* a])
+  ; similarly for tag range: 6.1280 .. 6.1400 inclusive
+  / #6.102([uint, [* a]])
+
+redeemer = [ tag: redeemer_tag, index: uint, data: plutus_data, ex_units: ex_units ]
+redeemer_tag =
+    0 ; inputTag "Spend"
+  / 1 ; mintTag  "Mint"
+  / 2 ; certTag  "Cert"
+  / 3 ; wdrlTag  "Reward"
+ex_units = [mem: uint, steps: uint]
+
+ex_unit_prices =
+  [ mem_price: sub_coin, step_price: sub_coin ]
+
+language = 0 ; Plutus v1
+         / 1 ; Plutus v2
+
+costmdls = { * language => cost_model }
+
+cost_model = [ 166*166 int ]
```

Comment:
```suggestion
costmdls = { ? 0 : [ 166*166 int ]
           , ? 1 : [ (166+x)*(166+x) int ] ; where x accounts for new V2 cost model params
           }
```

Reply:
@goolord - I'm going to remove this suggestion for now since we don't yet know what `x` is, but we can expect to add this back in pretty soon.